sdk/test/IntegrationTests/Tests/General.cs

```diff
@@ -201,26 +201,6 @@ public void TestSerializingExceptions()
             }
         }
 
-        [TestMethod]
-        [TestCategory("General")]
-        public void TestAnonymousCredentialsGetThroughPipeline()

```

Comment:
This test case isn't valid anymore since `AnonymousAWSCredentials` cannot be used for actions that require credentials.